.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: stable
+    rev: 19.10b0
```

Comment:
Has to be the same as in extra_requirements, otherwise there seems to be a formatting conflict.

Reply:
(move to seperate PR)